lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -1387,26 +1483,31 @@ class MongoClientInterface {
                 const obj = doc.value;
                 const objMetadata = new ObjectMD(obj.value);
                 objMetadata.setOriginOp('s3:ObjectRemoved:Delete');
-                objMetadata.setDeleted(true);
+                // Not supported in 7.x
+                // objMetadata.setDeleted(true);
```

Comment:
do you mean S3C 7.x ?

this code is used exclusively for mongo, which is used only in 8.x ; and this is javascript, so no type checking : i.e. no reason to remove this line (and possibly have an issue on 8)